contractcourt/htlc_success_resolver.go

```diff
@@ -185,13 +188,19 @@ func (h *htlcSuccessResolver) broadcastSuccessTx() (*wire.OutPoint, error) {
 	//
 	// TODO(roasbeef): after changing sighashes send to tx bundler
 	label := labels.MakeLabel(
-		labels.LabelTypeChannelClose, &h.ShortChanID,
+		labels.LabelTypeSweepTransaction, nil,
 	)
 	err := h.PublishTx(h.htlcResolution.SignedSuccessTx, label)
 	if err != nil {
 		return nil, err
 	}
 
+	// Now that we have broadcasted the second-level success transaction,
+	// we report the final outpoint that was swept.
```

Comment:
nit: just note that this may be because we update/fee bump our batched sweep for some context?

Reply:
Good suggestion for better understanding what is going on here. I have added more clarification as a comment!

Reply:
Nice, thanks!